app/imports/ui/components/NavBar.jsx

```diff
@@ -39,6 +74,7 @@ class NavBar extends React.Component {
           )}
         </Menu.Item>
       </Menu>
+>>>>>>> Stashed changes
```

Comment:
Looks like your stashed changes are in here somehow. Something may have gone wrong in the merge/rebase. All the content above is contained twice, or is GitHub rendering this incorrectly.